docs: add tenantId parameter documentation to liveblocks-node API reference (#2830)
liveblocks/liveblocks: docs/pages/api-reference/liveblocks-node.mdx

These are arbitrary identifiers that make sense to your app, and that you can
refer to in the [Permissions REST API][] when assigning group permissions.

`tenantId` (optional) is the tenant for this user, will be set to `default` if not provided.

`userInfo` (optional) is any custom JSON value, which you can use to attach
static metadata to this user's session. This will be publicly visible to all

> other people in the room. Useful for metadata like the user's full name, or

Reviewer: ```suggestion
other people in the room. Useful for metadata like the user’s full name, or
```

    },
  },

>   // Optional, filter for a specific tenant

Reviewer: ```suggestion
  // Optional, authenticate this user on a specific tenant
```

  const { data: nextPage } = await liveblocks.getUserRoomSubscriptionSettings({
    userId: "steven@example.com",
    startingAfter: nextCursor,

>     tenantId: "acme-corp",

Reviewer: ```suggestion
```
